src/DiseaseParm.H

```diff
@@ -43,7 +43,12 @@ struct DiseaseParm
     /*! Transmission probabilities of children for each age group (<5, 5-17, 18-29, 30-64, 65+) for neighborhood cluster */
     amrex::Real xmit_nc_child[5] = {amrex::Real(.15*pHCC0), amrex::Real(.15*pHCC1), amrex::Real(.08*pHC), amrex::Real(.08*pHC), amrex::Real(.08*pHC)};
     /*! Work transmission probabilities for each age group (<5, 5-17, 18-29, 30-64, 65+) */
+    /* adding hypothetical transmission for now for work interaction, if a_j is older, will transmission be higher?*/
     amrex::Real xmit_work = amrex::Real(0.115*pWO);
+    amrex::Real xmit_work_diff_age = amrex::Real(0.090*pWO);
+    amrex::Real xmit_work_diff_group = amrex::Real(0.075*pWO);
+    amrex::Real xmit_work_diff_group_diff_age = amrex::Real(0.060*pWO);
+
```

Comment:
Do these lines belong to this PR? Or is this from the work you are doing with improving workplace interactions?

Reply:
true. Ill remove them